Source/CsDebugScript.Engine/Engine/Debuggers/DbgEngDll.cs

```diff
@@ -1500,14 +1823,7 @@ private struct KDHELP64
-        /// This context may be modified, so do not pass a context record that should not be modified.</param>
-        /// <param name="ReadMemoryRoutine">A callback routine that provides memory read services.When the StackWalkEx function needs to read memory from the process's address space, the ReadProcessMemoryProc64 callback is used.
-        /// If this parameter is NULL, then the function uses a default routine.In this case, the hProcess parameter must be a valid process handle.
-        /// If this parameter is not NULL, the application should implement and register a symbol handler callback function that handles CBA_READ_MEMORY.</param>
-        /// <param name="FunctionTableAccessRoutine">A callback routine that provides access to the run-time function table for the process.This parameter is required because the StackWalkEx function does not have access to the process's run-time function table. For more information, see FunctionTableAccessProc64.
-        /// The symbol handler provides functions that load and access the run-time table. If these functions are used, then SymFunctionTableAccess64 can be passed as a valid parameter.</param>
+        /// <param name="StackFrame">A pointer to a STACKFRAME_EX structure. This structure receives information for the next frame, if the function call succeeds.</param> /// <param name="ContextRecord">A pointer to a CONTEXT structure.This parameter is required only when the MachineType parameter is not IMAGE_FILE_MACHINE_I386. However, it is recommended that this parameter contain a valid context record. This allows StackWalkEx to handle a greater variety of situations.  /// This context may be modified, so do not pass a context record that should not be modified.</param> /// <param name="ReadMemoryRoutine">A callback routine that provides memory read services.When the StackWalkEx function needs to read memory from the process's address space, the ReadProcessMemoryProc64 callback is used.  /// If this parameter is NULL, then the function uses a default routine.In this case, the hProcess parameter must be a valid process handle.  /// If this parameter is not NULL, the application should implement and register a symbol handler callback function that handles CBA_READ_MEMORY.</param> /// <param name="FunctionTableAccessRoutine">A callback routine that provides access to the run-time function table for the process.This parameter is required because the StackWalkEx function does not have access to the process's run-time function table. For more information, see FunctionTableAccessProc64.  /// The symbol handler provides functions that load and access the run-time table. If these functions are used, then SymFunctionTableAccess64 can be passed as a valid parameter.</param>
         /// <param name="GetModuleBaseRoutine">A callback routine that provides a module base for any given virtual address.This parameter is required.For more information, see GetModuleBaseProc64.
```

Comment:
Yes, not sure how but yes.

---
In reply to: [69632158](https://github.com/southpolenator/WinDbgCs/pull/8#discussion_r69632158) [](ancestors = 69632158)